src/main/java/com/github/nkzawa/engineio/client/transports/WebSocket.java

```diff
@@ -139,7 +169,11 @@ protected void onClose() {
 
     protected void doClose() {
         if (this.ws != null) {
-            this.ws.close();
+            try {
+                this.ws.close(1000, "");
+            } catch (IOException e) {
+                e.printStackTrace();
```

Comment:
When does this error happens?  I think we can ignore the error if it happens only when the connection is already closed, otherwise we should do something?  Anyway calling `printingStackTrace` would not be good idea here.